suite-common/trading/src/utils.ts

```diff
@@ -427,3 +427,62 @@ export const getTokenSelectableNetworks = (
                 TOKEN_SELECT_SELECTABLE_NETWORKS.includes(n.symbol),
         )
         .map(n => n.symbol);
+
+// reserve defined in networksConfig.ts
+// applies to the native token only
+export const getNetworkReserve = (
+    symbol: NetworkSymbol,
+    contractAddress: string | undefined | null,
+    isEnabled?: boolean,
+) => {
+    if ((!!contractAddress && contractAddress !== CONTRACT_ADDRESS_FOR_NATIVE_TOKEN) || !isEnabled)
```

Comment:
`CONTRACT_ADDRESS_FOR_NATIVE_TOKEN` can this even happen? trading backend uses but Suite does not